homework/Roman/task_01/task_01.py

```diff
@@ -0,0 +1,58 @@
+def convert_n_to_m(x, n: int, m: int):
+    """Calculates representation of x in base m.
+
+    in case x can't be represented as number or row,
+    or x can't be represented in base n returns False
+
+    Arguments:
+    x -- an input, integer or string with base n
+    n -- base for an input x, where 1 <= n <= 36
+    m -- base for returned value, where 1 <= m <= 36
+
+    """
+
+    if isinstance(x, float):
+        return 'False'
+
+    if isinstance(x, list):
+        in_string = str(x)[1:-1]
+    else:
+        in_string = str(x)
+
+    dict_key = list(map(chr, range(48, 58)))
+    dict_key.extend(list(map(chr, range(65, 91))))
+    dict_val = list(map(int, range(0, 37)))
+
+    forward_dict = dict(zip(dict_key, dict_val))
```

Comment:
dict comprehension is faster. Use `enumerate` function to build the forward_dict and reverse_dict.

Reply:
Replaced